Resources/views/CRUD/base_edit_form.html.twig

```diff
@@ -33,7 +33,7 @@
                         <div class="nav-tabs-custom">
                             <ul class="nav nav-tabs" role="tablist">
                                 {% for name, form_tab in admin.formtabs %}
-                                    <li{% if loop.index == 1 %} class="active"{% endif %}><a href="#tab_{{ loop.index }}" data-toggle="tab"><span class="glyphicon glyphicon-exclamation-sign has-errors hide"></span> {{ admin.trans(name, {}, form_tab.translation_domain) }}</a></li>
+                                    <li{% if loop.index == 1 %} class="active"{% endif %}><a href="#tab_{{ loop.index }}" data-toggle="tab"><span class="glyphicon glyphicon-exclamation-sign has-errors hide"></span> {{ admin.trans(form_tab.label, {}, form_tab.translation_domain) }}</a></li>
```

Comment:
This is explained in the PR comment : currently, when doing `->tab('field')`, the label translation id is `field`. This is not uniform with `->add('field')`, which label translation id is `form.label_field`.